### DISCUSSION TITLE
How to use DDP on Sagemaker using Pytorch Lightning?

### DISCUSSION THREAD

**Discussion by shubhamwagh** [Lightning-AI/pytorch-lightning#6852] [DDP / multi-GPU / multi-node]
I see that Pytorch Lightning (PL) provides support for multi-node training.
How to use PL trainer to launch multiple Sagemaker instances (nodes) for training? What explicit settings would I need in Sagemaker and PL so as to able to train in DDP fashion.

Thanks.

**Comment by celsofranssa** [comment]
Do you have any directions here?
